packages/demos-babylon/src/helloWorldDemo.js

```diff
@@ -27,12 +28,11 @@ async function createScene() {
   // "Cristine", "Fiona", "Grace", "Maya", "Jay", "Luke", "Preston", "Wes"
   const characterId = 'Cristine';
```

Comment:
This name is not consistently used on html file.

Reply:
Can you ellaborate? I'm intentionally using different host characters in the different demos so that customers get to see many of the hosts in action.

Reply:
This is just related to the previous comment I left about Christine or Cristine.